### ISSUE TITLE
Compiling on Fedora 30 x64

### ISSUE THREAD

**Issue by noriegafam** [Laidout/laidout#18]
I am able to run the AppImage, however I don't know integrate it as in your video youtube "Booklets in Scribus imposed by Laidout"
On the other hand when I get errors trying to compile. Thanks for any help you can provide!
The only version of gegl-devel available for fc30 is:
Installed:
  gegl04-devel-0.4.16-4.fc30.x86_64        babl-devel-0.1.66-1.fc30.x86_64        json-glib-devel-1.4.4-2.fc30.x86_64

./configure
Checking for x11......ok
Checking for xext......ok
Checking for freetype2......ok
Checking for libssl......ok
Checking for imlib2......ok
Checking for cairo......ok
Checking for harfbuzz......ok
Checking for gegl-0.3......You may need to install the development package of gegl-0.3.
Checking for ftgl......ok
Checking for GraphicsMagick++......ok
Checking for sqlite3......ok
Checking for readline......ok
Checking for cups......ok

Too many problems to proceed!
Use --force to override. You may have to adjust CPPFLAGS and LDFLAGS in Makefiles,
or pass in includes with --extra-cppflags.

**Comment by tomlechner** [comment]
For compiling, try with `./configure --gegl-version=gegl-0.4`

About integrating with Scribus, hmm, I guess with the AppImage, there's not direct access to the python script Scribus needs to call. The script you want is here:
https://github.com/Laidout/laidout/blob/master/coop/scribus/scribus-call-laidout-to-impose-gui.py
This is what you would call through Scribus's "Execute Script" command.

I really need to make an updated Scribus+Laidout video. That one's very old, and I haven't tested against current Scribus nearly enough. If things don't work, please let me know!

**Comment by noriegafam** [comment]
Thank you! At least I was able to compile it successfully with the --gegl-version=gegl-04 option. LD complained about not finding -ltiff, but installing libtiff-devel took care of that. 
Unfortunately, I was not able to get the script working even changing the line for the location of laidout 
whereis laidout
laidout: /usr/local/bin/laidout.
The script kept complaining about changing the line: laidoutexecutable="/usr/local/bin/laidout".
The real weird thing is the program pupped up a window to browse the file system to give me a chance to find laidout executable but the folders showing in the little window did not match what my local file manager will show. I was never able to see or find the "local" directory. 
Maybe I compiled it the wrong way. I don't know. I did:
 sudo ./configure --gegl-version=gegl-0.4
make
sudo make install
